src/client.js

```diff
@@ -447,6 +449,34 @@ Client.prototype.compileCustomHighlights = function () {
 	);
 };
 
+Client.prototype.compileCustomHighlightExceptions = function () {
```

Comment:
Would be nice to have a single function so it doesn't duplicate all the code. You can pass `client.config.clientSettings.highlightExceptions` into it, and make it return null or the compiled regex.